docs/carrier.md

```diff
@@ -102,6 +102,11 @@ From the **Carriers** page, you can perform several management operations.
     * **Tags** and **TOML:** Not applicable to carriers.
     * **Ext.Accounting ID:** This is used to fill in work in an external accounting field to correlate between Connexcs and the external accountancy system.
     * **Call Recording Retention Days:** How long the customer wants to keep the recorded calls.
+    * **Gateway Failover:** This feature allows you to swtich to another IP if the call(s) doesn't pass through or fails on the first IP. It has 4 options:
+        * *None*: The call will not switch to any other IP.
+        * *Failover 1*: The call will only to switch to 1 of the IP's that you've configured in the Auth.
+        * *Failover 2*: The call will only to switch to 2 of the IP's that you've configured in the Auth.
+        * *Failover 3*: The call will only to switch to 3 of the IP's that you've configured in the Auth section.
 
```

Comment:
Add something like this    Gateway Failover is a feature that enables automatic switching between multiple IP addresses configured within a carrier's authentication settings. By default, when a call reaches this carrier, it is attempted on one of the IPs configured in the carrier's authentication settings. If the call fails, it does not fail over to a different IP within the same carrier. However, enabling Gateway Failover allows the call to fail over to a different IP within the same carrier.

Options:
Gateway Failover typically offers several options that determine the behavior when a call fails on the first IP:

None: No failover occurs.
Failover 1: After a call fails on the first IP, the system attempts to route the call through one alternative IP configured in the authentication settings.
Failover 2: After a call fails on the first IP, the system attempts to route the call through up to two alternative IPs.
Failover 3: After a call fails on the first IP, the system attempts to route the call through up to three alternative IPs.

Reply:
done